provider/export/tests/test-options.rs

```diff
@@ -204,27 +208,16 @@ fn all_runtime() {
     let locales = [
         "ar",
         // "ar-EG", (same as 'ar')
-        "ar-EG-u-nu-latn", // (same as 'ar-u-nu-latn' but DIFFERENT than 'ar-EG')
-        "ar-u-nu-latn",
-        "bn",
-        "bn-u-nu-latn",
-        "ccp",
-        "ccp-u-nu-latn",
+        // "ar-EG/latn", (same as 'ar/latn')
+        "ar/latn", "bn", "bn/latn", "ccp", "ccp/latn",
```

Comment:
Nit: Please restore the previous line wrapping with one item per line. It used to work and I don't know why it would now wrap differently.

Reply:
it used to be in consts, that's the only difference I see. rustfmt does this